huaweicloud/services/acceptance/acceptance.go

```diff
@@ -1448,3 +1450,11 @@ func TestAccPreCheckVPCEPServiceId(t *testing.T) {
 		t.Skip("HW_VPCEP_SERVICE_ID must be set for the acceptance test")
 	}
 }
+
+// lintignore:AT003
+func TestAccPreCheckLTSCCEAccess(t *testing.T) {
+	if HW_LTS_LOG_STREAM_ID == "" || HW_LTS_LOG_GROUP_ID == "" || HW_LTS_CCE_CLUSTER_ID == "" || HW_LTS_HOST_GROUP_ID == "" {
+		t.Skip("The cce access config of HW_LTS_LOG_STREAM_ID, HW_LTS_LOG_GROUP_ID, HW_LTS_CCE_CLUSTER_ID" +
+			" and HW_LTS_HOST_GROUP_ID must be set for the acceptance test")
+	}
+}
```

Comment:
```suggestion
// lintignore:AT003
func TestAccPreCheckLTSCCEAccess(t *testing.T) {
	if HW_LTS_HOST_GROUP_ID == "" || HW_LTS_CCE_CLUSTER_ID == "" {
		t.Skip("The cce access config of HW_LTS_HOST_GROUP_ID and HW_LTS_CCE_CLUSTER_ID must be set for the acceptance test")
	}
}
```
Why are the other two environment variables verified together instead of using their corresponding verification functions in the test case?

Reply:
done